flow-typed/box-ui-elements.js

```diff
@@ -542,33 +542,46 @@ type Translations = {
     onTranslate?: Function
 };
 
-type OpenWithIntegrations = {
-    default_app_integration?: AppIntegrationMiniItem,
+type OpenWithAPI = {
+    default_app_integration?: AppIntegrationAPIMiniItem,
     disabled_reasons?: Array<string>,
     icon?: string,
     is_disabled?: boolean,
-    items: Array<OpenWithIntegrationItem>,
+    items: Array<OpenWithAPIItem>,
     should_show_consent_popup?: boolean
 }
 
-type OpenWithIntegrationItem = {
-    app_integration: AppIntegrationMiniItem | AppIntegrationItem,
+type OpenWithAPIItem = {
```

Comment:
These are now pure API responses. Objects that are manipulating the response are not typed until we create the well formed Integration object.